gui/src/hooks/countdown.ts

```diff
@@ -11,7 +11,7 @@ export function useCountdown({
 }) {
   const [isCounting, setIsCounting] = useState(false);
   const [timer, setDisplayTimer] = useState(0);
-  const countdownTimer = useRef<NodeJS.Timeout>();
+  const countdownTimer = useRef<number>();
```

Comment:
*technically incorrect*

Reply:
i could use `ReturnType<typeof setInterval>`, but apparently the migration to pnpm now correctly expects web code instead of node code

Reply:
interesting